common/deployment/setup/setup-jmeter-client-is.sh

```
@@ -33,7 +33,27 @@ alpnboot_dir="/opt/alpnboot"
 command="$script_dir/setup-jmeter-client.sh $@ \
-    -w http://search.maven.org/remotecontent?filepath=org/mortbay/jetty/alpn/alpn-boot/8.1.12.v20180117/alpn-boot-8.1.12.v20180117.jar \
+    -w https://repo1.maven.org/maven2/org/mortbay/jetty/alpn/alpn-boot/8.1.12.v20180117/alpn-boot-8.1.12.v20180117.jar \
     -o $alpnboot_dir/alpnboot.jar -j bzm-parallel"
 echo $command
-$command > /dev/null 2>&1
+jmeter_setup_log="/tmp/jmeter-client-setup.log"
+if ! $command > "$jmeter_setup_log" 2>&1; then
+    if grep -q "Unable to access jarfile.*cmdrunner-2.0.jar" "$jmeter_setup_log"; then
+        echo "Detected cmdrunner version mismatch. Applying fix..."
+        jmeter_home=$(find /home/ubuntu -maxdepth 1 -type d -name "apache-jmeter-*" | head -1)
+        latest_cmdrunner=$(ls "$jmeter_home/lib/cmdrunner-"*.jar 2>/dev/null | sort -V | tail -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)